src/aff_expr.jl

```diff
@@ -291,13 +291,17 @@ function AffExpr(m::Model, f::MOI.ScalarAffineFunction)
     aff.constant = f.constant
     return aff
 end
-function jump_function_type(::AbstractModel,
+function jump_function_type(::Model,
```

Comment:
Why does `jump_function` use `AbstractModel` and `jump_function_type` use `Model`? Is this explained anywhere?